src/openms/include/OpenMS/ANALYSIS/MAPMATCHING/QTClusterFinder.h

```diff
@@ -112,10 +113,24 @@ namespace OpenMS
 
     /// Map to store which grid features are next to which clusters
     typedef OpenMSBoost::unordered_map<
-              OpenMS::GridFeature*, std::vector<QTCluster*> > ElementMapping;
+              OpenMS::GridFeature*, std::unordered_set<Size> > ElementMapping;
+
+    /// Handles for the clusters in the index
+    typedef boost::heap::fibonacci_heap<QTCluster>::handle_type Handle;
+
+    /// Heap to efficiently find the best clusters
+    typedef boost::heap::fibonacci_heap<QTCluster> Heap;
+
+    /// Map of neighbors of a cluster
+    typedef std::pair<double, GridFeature*> NeighborPairType;
```

Comment:
can you think of a better name?
`Type` does not add anything. This thing is a type by definition (typedef!). And the double is a Score, right?

See https://youtu.be/MBRoCdtZOYg?t=1721 (the whole talk is worth looking at)

Reply:
We didn't come up with this and I don't know for sure what the double is. I think it's the distance to the cluster center. Just `Neighbor` would probably be enough.